Makefile

```diff
@@ -217,6 +218,26 @@ local-cloud-build: LOCAL_CLOUD_BUILD_PUSH = # --push
 local-cloud-build: gcloud
 	cloud-build-local --config=cloudbuild.yaml --dryrun=false $(LOCAL_CLOUD_BUILD_PUSH) --substitutions SHORT_SHA=$(VERSION_SUFFIX),_GCB_POST_SUBMIT=$(_GCB_POST_SUBMIT),_GCB_LATEST_VERSION=$(_GCB_LATEST_VERSION),BRANCH_NAME=$(BRANCH_NAME) .
 
+# Below should match push-images
+retag-images: retag-service-images retag-example-images retag-tool-images
+
+retag-service-images: retag-backend-image retag-frontend-image retag-mmlogic-image retag-minimatch-image retag-synchronizer-image retag-swaggerui-image
+retag-example-images: retag-demo-images retag-mmf-example-images retag-evaluator-example-images
+retag-demo-images: retag-mmf-go-soloduel-image retag-demo-image
+retag-mmf-example-images: retag-mmf-go-soloduel-image retag-mmf-go-pool-image
```

Comment:
regating this one twice?

Reply:
They are dependencies of a meta target it won't get retagged twice.